Add Website working group (#10)
django/dsf-working-groups: active/website.md

# Website Working Group

## Scope of responsibilities

This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.

The duties of the working group are:
- Introduce new features on the website
- Maintain and monitor the website 

> - Ensure that information on the website is accurate 

Reviewer: ```suggestion
```

Based on group discussions today, would recommend taking this out so it’s clear the group is meant to focus on features and code, and so we don’t have to figure out how to delegate access to the Django admin.

Reviewer 2: I agree

- Help to make the website accessible to all

### Delegated responsibilities:
- Members triage the project’s issues and pull requests.
- Member maintain and monitor the website including updating versions.
- Mentor new contributors to the website.
- Run or support djangoproject.com sessions in DjangoCon sprints.
- Chair, Co-Chair and Board Liaison can sign off on new features.

## Initial membership

- Chair: Sarah Abderemane
- Co-Chair: Saptak Sengupta
- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): Sarah Abderemane
- Other members:
    - Eric Sherman
    - Mark Walker
    - Jason Judkins
    - Paolo Melchiorre
    - Sanyam Khurana
    - Tobias McNulty

>     - Ron Maravanyika

Reviewer 3: > Are all these names still valid? Or we have to update them?

Yes they should be valid but I think some members are missing from the last messages: @Stormheg, @sakhawy and folks from DjangoCon US like @jacklinke @jbeimler I believe (I didn't see more names but maybe missing some here) 

> People in this list are still interested in joining the WG?

I think almost everyone were at DjangoCon US and mentioned in the thread except Eric Sherman.

> Do they already participate in the website maintenance?

I'm not sure but it's something I wanted to see when everyone shows up during a call or asynchronously.

> Can we consider all the people in the initial list on trial, and then make evaluations of the effective participation after 1 year?

Sounds reasonable for me.

>     - Ron Maravanyika

Author: @pauloxnet all the names should be valid. If folks want to join they just need to confirm that they want to join the group as @alexgmin has done. Earlier we wanted to include every contributor but there was a suggestion that folks should confirm their interest, so everyone on that list are individuals that have confirmed. We also had a suggestion to have a cap on the number of members we can have, personally l don't agree with that.

>     - Ron Maravanyika

Author: > > Are all these names still valid? Or we have to update them?
> 
> Yes they should be valid but I think some members are missing from the last messages: @Stormheg, @sakhawy and folks from DjangoCon US like @jacklinke @jbeimler I believe (I didn't see more names but maybe missing some here)
> 
> > People in this list are still interested in joining the WG?
> 
> I think almost everyone were at DjangoCon US and mentioned in the thread except Eric Sherman.
> 
> > Do they already participate in the website maintenance?
> 
> I'm not sure but it's something I wanted to see when everyone shows up during a call or asynchronously.
> 
> > Can we consider all the people in the initial list on trial, and then make evaluations of the effective participation after 1 year?
> 
> Sounds reasonable for me.

@sabderemane if the new members have confirmed l'm happy to add them. Have they confirmed?

>     - Ron Maravanyika

Reviewer 4: @ronnzw @sabderemane I'm happy to be involved with the working group. Haven't done any development on djangoproject.com yet, but I'm willing to familiarize myself with it and help out where I can.

>     - Ron Maravanyika

Author: > @ronnzw @sabderemane I'm happy to be involved with the working group. Haven't done any development on djangoproject.com yet, but I'm willing to familiarize myself with it and help out where I can.

Is Storm Heg the correct name to add?

>     - Ron Maravanyika

Reviewer 4: @ronnzw yes 👍

> 
> 
> 
> ## Future membership

Reviewer 2: ```suggestion

## Future membership
```

### Who is eligible to join? Any volunteer, or are there specific requirements?

Members must have interest in Django and should be able to work with Django. Members must be well versed with the process of contributing to **djangoproject.com** or at least willing to be guide. We welcome all experience levels, we also welcome first time contributors. 

### How do people who want to join sign up / volunteer / express interest?

> Individuals can express interest by emailing to the working group mailing list at `website-wg@djangoproject.com` 

Reviewer 2: ```suggestion
Individuals can express interest by opening a PR to this working group repository, using a template, adding their names in the list of members.
```
Let's not overcomplicate things, and use GitHub for everything, so we are more transparent about candidacy and we avoid using other software and platforms, avoiding also privacy issues.

Reviewer 3: If we are going in this process, I believe we should have a template form and make sure people answer to each questions and not erasing them for whatever reason. (even tho I don't think this is something required to make the WG reviewed by the DSF board to start the working group)

Reviewer 5: [Github Issue Templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) are a really easy, convenient way of doing this.

Reviewer 2: > [Github Issue Templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) are a really easy, convenient way of doing this.

Yes, I think Django Commons has something like that and we can ask for help to set up Tim or others.
https://github.com/django-commons/membership/issues/new/choose

> ### How will decisions on adding/removing members be handled?

Reviewer: Missing removing members.

Reviewer 6: I think after the one year term, anyone who doesn't opt-in to continue being involved in the WG can be removed. (We can also do an opt-out instead of opt-in mechanism. I don't have a super strong feeling about either). Also, I feel, maybe violation of code of conduct can result in removal of a member?

Reviewer 5: > Also, I feel, maybe violation of code of conduct can result in removal of a member?

I think that's implied generally, the CoC-WG (and the Board) has the power to impose various sanctions, including removing from WGs if necessary. So you don't need to explicitly say it here.

Author: Resolving to keeping the one year opt-in option.

> Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members. The WG will vote for New Chair/Co-Chairs and decision to appoint will be based on gaining majority votes.

Reviewer 2: ```suggestion
Direct membership: new members may self-nominate opening a PR to this repository; the WG members will approve the PR (with a required quorum of 50% + 1) to approve new members, or comment the PR with motivation to deny, having at maximum 1 month to express their intent. The WG members will vote for New Chair/Co-Chairs and decision to appoint will be based on gaining majority votes, using the same PR flow.
```
As I said above , let's use one tool for everything.

Reviewer 5: Suggestion: have a smaller group be responsible for approving members - maybe just the chair and co-chair? Or them plus a couple of others?

Background is that the way DSF membership works right now involves a full board vote, similar to what you're proposing here, and it's a pain honestly. It takes a lot of time, and it means there's always a pretty long delay between application and approval. And, most of the time, it's pretty obvious, there are very very few non-obvious decisions. So I'd suggest you aim to streamline joining the group, with just a few people delegated to make quick decisions. You could always toss any difficult calls to the broader membership, so only the complex ones need take longer.

Reviewer 2: > Suggestion: have a smaller group be responsible for approving members - maybe just the chair and co-chair? Or them plus a couple of others?

Great suggestion. I updated my proposal of changes.

> Members join the group for one year term. At the end of this term, they need to opt into staying involved to keep being 
> a member of the group.

Reviewer 2: ```suggestion
Members join the group for one year term. At the end of this term, they need to opt into staying involved to keep being a member of the group.
```

> 
> 
> ## Budget

Reviewer 2: ```suggestion

## Budget
```

> No allocated budget

Reviewer 2: ```suggestion
No allocated budget.

Sums may be provided for specific activities (e.g. external consultancy, graphic work, use of platforms, ...) subject to approval by the DSF board.
```

> ## Comms

Reviewer 2: ```suggestion
## Communications
```
Let's avoid slang or short form in this type of document.

> - Private channel in the DSF slack

Reviewer 2: ```suggestion
- A channel in the Django discord server, accessible only to DSF mrmbers
```
It would be possible to use a subsection of the DSF-Members in the Discord server? It's already there, and it will help to interact with other WGs (eg. accessibility, social, ...)

Reviewer 2: > I'm confused, according Thibaud's answer in the thread below it's should be a DSF slack channel
Yes, you are right. I only realized that the Discord is already there. Using Slack will add, another tool to install and use in addition to Django Forum, Discord server, etc ... We risk spending a lot of time and focus only on checking all these tools and switching between them.
But, we already agreed on Slack so I'll close the suggestion.

> - A mailing list that we'll create, `website-wg@djangoproject.com`

Reviewer 2: ```suggestion
- A Django forum sub-section "Website" of Django Internals
```
We migrated developer and user mailing list to the Django Forum, so let's try to use the same platform for asynchronous communication (the preferred way as we can be in different time zones), but having the private Slack or Discord channel for synchronous communications in case of emergency, meetings or for faster interaction.

Reviewer: Sounds great, particularly considering that section already exists! [Website](https://forum.djangoproject.com/c/internals/website/27).

Reviewer: @ronnzw this is the last thread about discussion spaces that is still open.

Reviewer 7: I would rather just use the forum personally.

## Reporting

> We'll email a written report to the board every quarter.

Reviewer 2: ```suggestion

We'll create a changelog file in the repository with all changes, using the https://keepachangelog.com format.
```
Let's use the tools we already have to share changes.

Reviewer 2: > I like the idea but I don't think it will suits for everything ...
I agree. I updated my change suggestion.

# Website Working Group

## Scope of responsibilities

> This working group will manage Django's official website, **djangoproject.com** on behalf of the Django Software Foundation

Reviewer 5: You probably need to clarify "manage" a bit -- we already have an ops team (hi @django/ops-team) that does a few things website-related. Delineating the devision of labor here, at least roughly, would be a good idea.

Author: I need help to clarify this, I'm not sure what the @django/ops-team 's tasks are here.

Reviewer: I would recommend something like:

```suggestion
This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.
```

Deployments in particular feel to me like something that ought to be automated, rather than depending on any one team.

Author: Thank you both for your input here.

The duties of the working group are:
- Introduce new features on the website
- Maintain and monitor the website 
- Ensure that information on the website is accurate 
- Help to make the website accessible to all

> 

Reviewer 8: We also run or support djangoproject.com sessions in DjangoCon sprints.
https://github.com/django/djangoproject.com/pulls?q=is%3Apr+is%3Aclosed+label%3A%22DjangoCon+%F0%9F%A6%84%22

Author: @cgl thanks

Author: @thibaudcolas any objections to this?

Reviewer: No?

### Delegated responsibilities:

> - Members triage the project’s issues and pull requests.
> - Member maintain and monitor the website including updating versions
> - Mentor new contributors to the website
> - Run or support djangoproject.com sessions in DjangoCon sprints.
> - Chair, Co-Chair and Board Liaison can sign off on new features

Reviewer 8: ```suggestion
- Members triage the project’s issues and pull requests.
- Member maintain and monitor the website including updating versions.
- Mentor new contributors to the website.
- Run or support djangoproject.com sessions in DjangoCon sprints.
- Chair, Co-Chair and Board Liaison can sign off on new features.
```

## Initial membership

- Chair: TBA
- Co-Chair: TBA

> - Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): Thibaud Colas

Reviewer: I think this is premature. @sabderemane is a board member, she expressed an interest in helping with this too, a good two months before I did. @cgl is a board member who is already involved with the site’s maintenance and might also be keen to be Board Liaison, though she hasn’t said what she’d think of continuing this as a working group. The three of us (and any other interested board member) can figure it out.

```suggestion
- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBA
```

> - Other members:
>     - Jacob Kaplan-Moss

Reviewer: I’m not clear that Jacob said he was interested in joining this group. He started #2 but I think it was just an admin task for him?

```suggestion
- Other members:
```

Reviewer 5: Indeed, I've got enough on my plate, I can't take this on as well.

>     - Sarah Abderemane

Reviewer 3: ```suggestion
```

    - Eric Sherman
    - Mark Walker
    - Jason Judkins
    - Paolo Melchiorre
    - Sanyam Khurana
    - Tobias McNulty

>     - Ron Maravanyika

Reviewer 8: This is missing a few members from the current maintainers group.

Author: @cgl yes that is correct. This list is just a list of individuals who confirmed that they wanted to be part of the working group. Because there are some individuals who are part of the current maintainers group that don't want to be part of the working group as they won't be contributing going forward.

    - Ron Maravanyika

## Future membership

### Who is eligible to join? Any volunteer, or are there specific requirements?

Members must have interest in Django and should be able to work with Django. Most importantly members must be willing to adhere to Django's [Code of Conduct](https://www.djangoproject.com/conduct/) . Members must be well versed with the process of contributing to **djangoproject.com** or at least willing to be guide. We welcome all experience levels, we also welcome first time contributors. 

### How do people who want to join sign up / volunteer / express interest?

> TBA

Reviewer: Please fill this in, even with just a bad first draft.

### How will decisions on adding/removing members be handled?
Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members; the WG will directly vote on new Chair/Co-Chairs.

Members join the group for a 2 year term. At the end of this term, they need to opt into staying involved to keep being 

> a member of the group.

Reviewer 5: This is an interesting idea, I hadn't seen it before! What was the reasoning here? Is this something that maybe is a good enough idea we should adopt more broadly?

Author: As time goes on people get new commitments and it's not always easy to give feedback on their intentions, especially for people that contribute on multiple projects. Not sure if it applies to all working groups.

Reviewer: Could we have a shorter term? 2 years feels like a long time to me, if people don’t want to be involved anymore it’d be better for them to be able to head off after 6-12 months.

Author: i feel two years is ample time for this type of groups, maybe we can add that individuals can leave at anytime. The idea here is not to keep individuals that aren't interested in the group anymore but haven't found time to notify the team. This is not to stop members from leaving the group.

Reviewer 8: I also think we could have shorter terms.

## Budget

> TBA

Reviewer: Please fill this in, even with just a bad first draft.

Reviewer 5: My suggestion: for now, no allocated budget (there's already an operational budget for the site)

## Comms

> - A mailing list that we'll create, `website-wg@djangoproject.com`

Reviewer: Could you explain why you’re proposing a mailing list? They tend to run slow in my experience and are prone to spam. Django has a Discord server where we could have a private channel. The DSF has a Slack workspace. Could we use either of those?

Author: Well this is open to discussion, maintainers can decided where they want comms to be. I just remembered that l got involved on the site through the DSF individual members mailing list so i just assumed.

Reviewer 9: Is there a DSF slack?

Reviewer: I see we now have both the Slack channel and a mailing list. @ronnzw I think you need to define how each of those would be used, or if it’s open to discussion then explicitly write it here.

Keep in mind the more clear-cut the group charter is, the easier it is for people to consider joining and for the board to review. So if discussions need to happen, now is the best time to raise this question with interested members.

---

@jcjudkins yes, the DSF has a Slack Workspace for internal matters. Largely this has been used for paid employees, the board of directors, and a few working groups now.

Reviewer 10: IMHO slack works fine for the working group. Mailing list can be for discussion among larger audience.

Reviewer 3: @ronnzw I think this one is the only missing update from your part according Thibaud's message

Reviewer: I see "DSF slack" is now there so I’d just recommend removing the point about the mailing list and that’s it :)

Author: Is everyone happy with me removing the mailing list? Despite a mailing list being included in the WG guidelines?

Reviewer 6: I can see a mailing list being useful as a way of making announcements and interacting with the wider audience. But if we feel that might not be as necessary for Website WG, then removing the point about mailing list makes sense.

Reviewer: We discussed this at today’s DjangoCon US sprint meeting, with @jacklinke @tobiasmcnulty @pauloxnet @CuriousLearner @jcjudkins @bmispelon @thibaudcolas @jbeimler. Almost everyone in this group already uses Slack and will be happy there. No objections to having a mailing list in addition if this feels helpful.

## Reporting

> TBA

Reviewer 5: My suggestion: for this group, I think a quarterly report is fine (plus whatever adhoc questions for the board might come up in between)

# Website Working Group

## Scope of responsibilities

This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.

> The duties of the working group are:

Reviewer: From this description it’s unclear to me if this working group also works on the website’s contents, or just specific things that count as "features"?

For example if someone from our Security team wants to publish a post on the website’s blog – do they reach out to this working group, or elsewhere?

Author: I had asked for help on this particular item and you are the one who provided the wording that is on there. I'm happy if you can add on that you feel would make sense.

Reviewer: @ronnzw cool! Let‘s see what others think then.

Reviewer 3: [According this thread](https://github.com/django/dsf-working-groups/pull/10/files#r1469462925) and if no one disagree, we keep the fact that the content is not written by us, it should be only specific things that count as "features" or things.

Reviewer: Cool!

- Introduce new features on the website
- Maintain and monitor the website 
- Ensure that information on the website is accurate 
- Help to make the website accessible to all

> ### Delegated responsibilities:

Reviewer: I would like to see two additions:

- Who has the right in the group to merge pull requests? Is it "chair / co-chair / board liaison", or "everyone" or "no one"?
- Who has the right to publish new content on the site?

Reviewer 6: > Who has the right in the group to merge pull requests? Is it "chair / co-chair / board liaison", or "everyone" or "no one"?

I don't think "everyone" is a particularly safe option. If we want to make it "everyone", the vetting process of adding someone as a member need to be more precise to ensure that nothing bad gets merged. Also, there needs to be a process to re-evaluate permissions if anyone feels like they need to leave the WG.

But I feel like the `django-com-maintainers` team who already have permission and are actively maintaining the website currently should still have the permission (unless they themselves don't want the permissions anymore).

So here are few thoughts of  mine:
- We can either make the process of adding someone to the WG more strict and always have a limited number of folks as members. From a security point-of-view, I am not a fan of too many people having merge access.
- The other option that I can think of is letting "everyone" have merge access, but making the branch protection rules stronger. For example, at least 2 members need to approve the PR fo it to be merged, and only the chair/co-chair/board liaison may be able to override it, in case of any emergency requirement.
- The third option I can think of is segregating members into Maintainers and Members, where Maintainers have merge access, but Members can triage and stuff.

Reviewer 3: > The third option I can think of is segregating members into Maintainers and Members, where Maintainers have merge access, but Members can triage and stuff.

IMO, the third option seems to be a good one, at least maintainers is not everyone and we don't have to restrict the access to be part of the working group. We can based the list on the current members who are part of the `django-com-maintainers` and add more members in the future if needed. We will need to define how to add a maintainer to the Maintainer team but I think it's not a prerequisite to kickstart this WG.

Reviewer 3: > I like the idea of Website WG not having permission to edit content (unless there are crossovers with other teams of course)

I agree on this, as it's replacement of the initial maintainers team, they actually don't have the write to publish content. 
I find quite interesting the idea of the editorial WG team, I'm not sure as a working group but this is a separate topic I will see as board member. 

> Given we have a board liaison who I think already have permission to publish content being part of the Board, that should suffice in case there is a need to intervene.

Yes 💯

Reviewer: Sounds good to me too! Agree it’s not a blocker to starting the WG since we already have a maintainers team currently.

Reviewer: Permissions /  access to resources:

- Website deployment
- Ping via `@django/ops-team` on the pull request for deployment.
- Merging: initial onboarding phase where only Chair and Co-Chair have permissions to merge. After onboarding of 3-6 months, other group members who have proven their expertise will be granted merge permissions.

Ops team will retain access, and in case there are site infrastructure issues, would be able to restrict permissions.

--- 

Other resources that are relevant but could be done ad-hoc:

- Stripe
- Sentry
- reCAPTCHA
- (Trac)
- Transifex

Reviewer 11: Sentry access to the (majority of) the group would be a great benefit. Maybe some of the group, chair/co-chair, having slightly different permission and having a sentry team defined.

Reviewer 7: I think this still needs to be addressed.

> - Chair, Co-chair and Board Liaison can make final decision of  sections of the website that might not conform with DSF's CoC.

Reviewer: ```suggestion
```

If there were CoC-related problems on the site, the final decision would be for the CoC committee / working group.

> - Members can work on bugs

Reviewer: ```suggestion
```

I believe anyone can contribute to the site as far as working on bugs, so we shouldn’t list this here unless we were intending to remove this option from people outside the group.

Author: For me the key wording is **can** meaning it's not only group members that can contribute.

Reviewer: @ronnzw yes, agreed, though I still think my suggestion stands. This section is about "Delegated responsibilities", which are things that the DSF _Board_ delegates to the group. So though fixing bugs can be done by this group, it’d be more meaningful for the charter to focus on things that are bigger responsibilities.

Author: @thibaudcolas makes sense.

> - Members can tag,label and respond to issues.

Reviewer: ```suggestion
- Members triage the project’s issues and pull requests.
```

Make it broader. "tag" and "label" is the same thing in GitHub?

> - Member can maintain and monitor the website including updating versions

Reviewer: ```suggestion
- Member maintain and monitor the website including updating versions
```

In my opinion if this is one of the group’s responsibilities then it should be clear they’re responsible for this.

- Mentor new contributors to the website

> - Chair, Co-Chair and Board Liaison can sign off on new features

Reviewer: ```suggestion
- Chair, Co-Chair and Board Liaison sign off on new features
```

## Initial membership

- Chair: TBA
- Co-Chair: TBA

> - Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBA

Reviewer: This needs filling in ahead of a board review.

Reviewer 3: I'm happy to be Co-chair if possible

    - Eric Sherman
    - Mark Walker
    - Jason Judkins
    - Paolo Melchiorre
    - Sanyam Khurana
    - Tobias McNulty
    - Ron Maravanyika

>     - Alex Gómez

Reviewer: ```suggestion
    - Alex Gómez
    - Storm Heg
```

>     - alexgmin Alex

Reviewer 12: Sorry didn't realize my github account didn't have my surname. Please change it to Alex Gómez.

    - Ron Maravanyika

## Future membership

### Who is eligible to join? Any volunteer, or are there specific requirements?

> Members must have interest in Django and should be able to work with Django. Most importantly members must be willing to adhere to Django's [Code of Conduct](https://www.djangoproject.com/conduct/) . Members must be well versed with the process of contributing to **djangoproject.com** or at least willing to be guide. We welcome all experience levels, we also welcome first time contributors. 

Reviewer: ```suggestion
Members must have interest in Django and should be able to work with Django. Members must be well versed with the process of contributing to **djangoproject.com** or at least willing to be guide. We welcome all experience levels, we also welcome first time contributors. 
```

Following the code of conduct is a fundamental requirement of all Django community interactions so doesn’t need calling out.

Reviewer: This website is pretty crucial to the Django community so I’d recommend further requirements. Possibly a requirement to be a DSF Individual Member.

Author: Won't this block new community members?

### How do people who want to join sign up / volunteer / express interest?

> Individuals can express interest by emailing to the working group mailing list at `website-wg@djangoproject.com` 

Reviewer: Could we set up a Google Form instead? From my experience with DSF mailing lists, it’s 99% spam that we receive on those emails.

It’d be nice for you to define here what kind of information people should provide when applying, rather than just expect applicants will figure it out.

Author: @django/djangoproject-com-maintainters what questions must we ask if we are to build the form?

Reviewer: I’d recommend "has to be a DSF member" as this is a clearly-defined status.

Author: I second the suggestion to be a DSF member to join.

Reviewer 6: I agree with "has to be a DSF member"

Reviewer 13: @thibaudcolas I discovered yesterday during the WG conversations that I am *not* yet a DSF member, but I have now been nominated. If that nomination goes through, and if this proposal is not yet finalized by that time, could you add me to the initial Working Group membership, please? Thank you!

Reviewer 5: > Members must be well versed with the process of contributing to **djangoproject.com**

So, this rules out new people - people who might be excited about helping but don't have experience and want to be taught/coached. That's at odds with 

> We welcome all experience levels, we also welcome first time contributors. 

---

Along similar lines, I'm not sure I agree with requiring DSF membership. That's again at odds with being open to all experience levels; DSF Members will, almost definitionally, have some experience already. 

I see contributing to the website as a really convenient "on ramp" to being involved in the Django community -- it's one of the few ways you could contribute to Django that involves merely writing a Django site, not doing something more complex. So I'd encourage keeping the requirements as minimal as possible.

Author: @jacobian seems most people would like individuals to be DSF members. But l total get your point, maybe we can refine this once we see the need to do so.

Author: > Could we set up a Google Form instead? From my experience with DSF mailing lists, it’s 99% spam that we receive on those emails.
> 
> It’d be nice for you to define here what kind of information people should provide when applying, rather than just expect applicants will figure it out.

We will be using PR for this.

### How will decisions on adding/removing members be handled?

> Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members; the WG will directly vote on new Chair/Co-Chairs.

Reviewer: How does the vote on Chair/Co-Chair work? Is it also 50%+1 or "whoever gets the most votes"?

Author: I will add this

# Website Working Group

## Scope of responsibilities

This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.

The duties of the working group are:
- Introduce new features on the website
- Maintain and monitor the website 

> - Help to make the website accessible to all

Reviewer 7: I wonder if this list and the delegated responsibilities could be a single list. Because I think the main responsibility is to make decisions on the direction of the website, and is a bit hidden in the second list under a bit more specific sounding "propose new features" which I think is selling it short a bit.

### Delegated responsibilities:
- Members propose, triage, analize, discuss and implement new features
- Members triage the project’s issues and pull requests.

> - Member maintain and monitor the website including updating versions.

Reviewer 7: ```suggestion
- Members maintain and monitor the website including updating versions.
```
